python/extensions/python.bzl

```diff
@@ -14,6 +14,6 @@
 
 "Python toolchain module extensions for use with bzlmod"
 
-load("//python/private/bzlmod:python.bzl", _python = "python")
+load("//python/private:python.bzl", _python = "python")
```

Comment:
I am wondering if we should have a separate `python/private/toolchain` or `python/private/interpreter` directory, which could have all of the `python` extension related things. The `pythons_hub.bzl` could be in that folder as well.

Reply:
SGTM. Probably an independent PR? It's a bigger change to discuss. Because we could also leave the python stuff where it is and move everything else out, which then makes it less urgent to change the Python toolchain or language rules.